loadable_apps/loadable_sample/wifiapp/binary_update.c

```diff
@@ -225,6 +246,7 @@ static void binary_update_download_new_binary(void)
 	/* Write the binary header. */
 	ret = write(write_fd, (FAR uint8_t *)&header_data, sizeof(binary_header_t));
 	if (ret != sizeof(binary_header_t)) {
+		ret = ERROR;
 		printf("Failed to write header: %d\n", ret);
```

Comment:
I think you'd better print the `ret` before assigning to `ERROR`.

Reply:
modified